Authentication/Podfile

```diff
@@ -4,4 +4,4 @@ xcodeproj 'Authentication.xcodeproj'
 platform :ios, '8.0'
 use_frameworks!
 
-pod 'AeroGearHttp', '0.2.0'
+pod 'AeroGearHttp', :git => "https://github.com/corinnekrych/aerogear-ios-http-1.git", :branch => "AGIOS-428.ios-8.3.migration"
```

Comment:
let's not forget to update to official branch

Similar needs to be done for all Podfile's in /demos/